### PR TITLE
docs(ios): adding license header

### PR DESCRIPTION
**Pull request by milos1290**

### CODE REVIEW

File: Leanplum-SDK/Classes/FileMD5Hash.c [hunk 1]
```diff
@@ -96,4 +112,4 @@ CFStringRef Leanplum_FileMD5HashCreateWithPath(CFStringRef filePath,
         CFRelease(fileURL);
     }
     return result;
-}
\ No newline at end of file
+}
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/FileMD5Hash.c]: Can you add empty line at the end?

File: Leanplum-SDK/Classes/LPFileManager.h [hunk 2]
```diff
@@ -3,8 +3,24 @@
 //  LeanplumTest
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LPFileManager.h]: Change to Leanplum

File: Leanplum-SDK/Classes/LPFileManager.m [hunk 3]
```diff
@@ -3,8 +3,24 @@
 //  LeanplumTest
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LPFileManager.m]: Change to Leanplum

File: Leanplum-SDK/Classes/LPKeychainWrapper.h [hunk 4]
```diff
@@ -11,4 +33,4 @@
     + (NSData *)cipherString:(NSString *)data withKey:(NSString*)cipherKey;
     + (NSString *)decipherString:(NSData *)data withKey:(NSString*)cipherKey;
     
-@end
\ No newline at end of file
+@end
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LPKeychainWrapper.h]: empty line at the end

File: Leanplum-SDK/Classes/LPMessageTemplates.m [hunk 5]
```diff
@@ -5,26 +5,26 @@
 //  Created by Andrew First on 9/12/13.
 //  Copyright 2013 Leanplum, Inc.
 //
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LPMessageTemplates.m]: Revert this file? looks the same but with spaces

File: Leanplum-SDK/Classes/Leanplum.m [hunk 6]
```diff
@@ -3,8 +3,24 @@
 //  Leanplum
 //
 //  Created by Andrew First on 4/30/12.
-//  Copyright (c) 2016 Leanplum. All rights reserved.
+//  Copyright (c) 2017 Leanplum. All rights reserved.
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/Leanplum.m]: Don't edit copyright year

Comment (gigasmile) [inline Leanplum-SDK/Classes/Leanplum.m]: Change to 2012

File: Leanplum-SDK/Classes/LeanplumInternal.h [hunk 7]
```diff
@@ -3,8 +3,24 @@
 //  Leanplum
 //
 //  Created by Andrew First on 4/30/15.
+//  Copyright (c) 2017 Leanplum. All rights reserved.
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LeanplumInternal.h]: 2015

File: Leanplum-SDK/Classes/LPUIAlert.m [hunk 8]
```diff
@@ -4,8 +4,24 @@
 //
 //  Created by Ryan Maxwell on 29/08/13.
 //  Copyright (c) 2013 Ryan Maxwell. All rights reserved.
-//  Copyright (c) 2015 Leanplum, Inc. All rights reserved.
+//  Copyright (c) 2015 Leanplum. All rights reserved.
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LPUIAlert.m]: 2013

File: Leanplum-SDK/Classes/LPUIAlert.h [hunk 9]
```diff
@@ -4,8 +4,24 @@
 //
 //  Created by Ryan Maxwell on 29/08/13.
 //  Copyright (c) 2013 Ryan Maxwell. All rights reserved.
-//  Copyright (c) 2015 Leanplum, Inc. All rights reserved.
+//  Copyright (c) 2015 Leanplum. All rights reserved.
```
Comment (gigasmile) [inline Leanplum-SDK/Classes/LPUIAlert.h]: 2013

File: Leanplum-SDK/Classes/FileMD5Hash.c (hunk 1 above)
Comment (milos1290) [inline Leanplum-SDK/Classes/FileMD5Hash.c]: hmm, it has empty line on my end.

File: Leanplum-SDK/Classes/LPKeychainWrapper.h (hunk 4 above)
Comment (milos1290) [inline Leanplum-SDK/Classes/LPKeychainWrapper.h]: same, it has empty line on my end.

File: Leanplum-SDK/Classes/LPMessageTemplates.m (hunk 5 above)
Comment (milos1290) [inline Leanplum-SDK/Classes/LPMessageTemplates.m]: Added space for consistency with other license headers.

File: Leanplum-SDK/Classes/Leanplum.m (hunk 6 above)
Comment (milos1290) [inline Leanplum-SDK/Classes/Leanplum.m]: both .h and .m ?

**Comment by benmarten** [comment]
its always `Leanplum, Inc`

**Pull request merged**